ipfs-pinning-service.yaml

```diff
@@ -1,11 +1,13 @@
 openapi: 3.0.0
 info:
   version: "0.0.3"
-  title: '(WIP) IPFS Pinning Service API'
+  title: 'IPFS Pinning Service API'
+  x-logo:
+    url: "https://bafybeidehxarrk54mkgyl5yxbgjzqilp6tkaz2or36jhq24n3rdtuven54.ipfs.dweb.link/?filename=ipfs-pinning-service.svg"
   description: "
   
   
-# Goal
+## About this spec
```

Comment:
@jessicaschilling mind quickly glancing at the new copy added in this PR? Thanks!